tests/components/calendar/test_init.py

```diff
@@ -419,35 +424,25 @@ async def test_list_events_service(
         blocking=True,
         return_response=True,
     )
-    assert response == {
-        "events": [
-            {
-                "start": "2023-06-22T05:00:00-06:00",
-                "end": "2023-06-22T06:00:00-06:00",
-                "summary": "Future Event",
-                "description": "Future Description",
-                "location": "Future Location",
-            }
-        ]
-    }
+    assert response == snapshot
```

Comment:
```suggestion
    assert response == snapshot(name="test_list_events_service")
```

Reply:
What does this change?

Reply:
this will create 1 snapshot for the whole test. So if for some reason the behaviour of one changes, the test will fail. 

(I am just trying to avoid people accepting snapshot changes as a change, without asking themselves, _should_ this change)

Reply:
Wait heh, this is not what I expected. I expected that syrupy would nicely create it by this name instead of the test name. Hmmmm...

Reply:
If this doesn't work lets remove it and work with the alternative:

```python
@freeze_time("2023-06-22 10:30:00+00:00")
@pytest.mark.parametrize(
    ("service", "expected"),
    [
        (SERVICE_GET_FORECAST, {
        "events": [
            {
                "start": "2023-06-22T05:00:00-06:00",
                "end": "2023-06-22T06:00:00-06:00",
                "summary": "Future Event",
                "description": "Future Description",
                "location": "Future Location",
            }
        ]
    }),
        (LEGACY_SERVICE_GET_FORECAST,{"calendar.test":
        "events": [
            {
                "start": "2023-06-22T05:00:00-06:00",
                "end": "2023-06-22T06:00:00-06:00",
                "summary": "Future Event",
                "description": "Future Description",
                "location": "Future Location",
            }
        ]
    }}),
    ],
)
@pytest.mark.parametrize(
    ("start_time", "end_time"),
    [
        ("2023-06-22T04:30:00-06:00", "2023-06-22T06:30:00-06:00"),
        ("2023-06-22T04:30:00", "2023-06-22T06:30:00"),
        ("2023-06-22T10:30:00Z", "2023-06-22T12:30:00Z"),
    ],
)
async def test_list_events_service(
    hass: HomeAssistant,
    set_time_zone: None,
    start_time: str,
    end_time: str,
    expected: dict,
) -> None:
```

Reply:
Agreed!

Reply:
Sorry for bothering you with it :)

Reply:
No problem. I am all for readable and maintainable code =)